easylist_adult/adult_specific_hide.txt

```diff
@@ -1143,6 +1150,7 @@ instantfap.com##a[href^="https://go.redgifcams.com/"]
 sexhd.pics##a[href^="https://go.stripchat.com/"]
 imgsen.com##a[href^="https://hardcoreincest.net/"]
 adultgamesworld.com##a[href^="https://joystick.tv/t/?t="]
+ryuugames.com##a[href^="https://l.erodatalabs.com/s/"]
```

Comment:
https://www.ryuugames.com/tropical-kiss-free-download/

![CleanShot 2025-02-28 at 20 09 29](https://github.com/user-attachments/assets/5bbedca3-8246-480b-a58c-cb1569df5d7d)

But considering [your comment](https://github.com/easylist/easylist/pull/21289/files#r1975606021) above, I guess it would be better to change it to
```adblock
ryuugames.com##.td-tags a[href^="https://l.erodatalabs.com/s/"]
```